### ISSUE TITLE
Fetching and underprediction and overprediction values

### ISSUE THREAD

**Issue by adidutt** [fairlearn/fairlearn#755]
Is there a way to fetch underprediction and overprediction values for each subgroup of a protected attribute (eg, for Gender - subgroups - Male, Female)?
![1](https://user-images.githubusercontent.com/53310020/115583605-08b8f580-a298-11eb-9f74-9e1be86f29bc.PNG)

**Comment by riedgar-ms** [comment]
I believe that those are the `_mean_underprediction` and `_mean_overprediction` metrics:
https://github.com/fairlearn/fairlearn/blob/2d795006c045bdaeab1a22cd1893d5c63d27f099/fairlearn/widget/_fairlearn_dashboard.py#L135
You can obviously call them directly (or hand them to a `MetricFrame`). I don't recall offhand why we put the underscore prefix on them.

**Comment by adidutt** [comment]
Thanks for the reply. I got the correct values through MetricFrame.
Earlier, I tried calling them directly like, for a protected attribute named 'Gender' I want the individual mean overprediction and the individual mean underprediction (for values 0 and 1).

print(_mean_overprediction(ground_truth, predicted_outcomes))
print(_mean_underprediction(ground_truth, predicted_outcomes))

But according to this code, the output would be for overall gender variable and not for individual values (0 and 1).
